tbx/core/models.py

```diff
@@ -130,7 +131,9 @@ class HomePagePartnerLogo(Orderable):
 # Home Page
 class HomePage(ColourThemeMixin, ContactMixin, SocialFields, NavigationFields, Page):
     template = "patterns/pages/home/home_page.html"
-    introduction = models.TextField(blank=True)
+    hero_heading_1 = models.CharField(max_length=255)
+    hero_heading_2 = models.CharField(max_length=255)
+    hero_introduction = RichTextField(blank=True, features=["bold", "italic", "link"])
```

Comment:
@SharmaineLim Have we considered componentizing this (e.g. creating a 'HomePageHeroBlock' and adding a new streamfield to this page?

Reply:
I haven't, but I'm also not quite sure what the benefit of converting this to a block would be aside from "less migrations if we choose to change this section in the future".

Reply:
> I'm also not quite sure what the benefit of converting this to a block would be

Like I hinted at before, it's mostly about componentisation... giving it it's own space to evolve (hero-specific validation etc), and reuse.

Reply:
Certainly not a requirement by any means... I just thought it was worth considering is all :)

Reply:
The nature of agile often means this kind of thing starts of as "We want to do this thing on a specific page", then often needs readapting/tweaking later to use in other places. 'Componentising by default' can help make that process a little easier.

Reply:
I can see where you're coming from; if this style of heading/hero gets used again in a different page, it'd currently be easier to copy the fields rather than converting the hero in the homepage into a block and reusing that component, which means we would end up with some duplication.

However, I don't see us reusing this style exactly on other pages because of the 100% employee-owned icon, and I think we'd end up with another version of this "component" anyway.

e.g. on the home page, we have the heading, the description AND the 100% employee-owned icon in the hero; but on other pages, we would probably only have the heading and the description